rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/OpenehrContributionController.java

```diff
@@ -153,7 +153,7 @@ public ResponseEntity getContribution(@ApiParam(value = REQ_OPENEHR_VERSION) @Re
                     respHeaders.setLocation(uri);
                     break;
                 case ETAG:
-                    respHeaders.setETag("\"" + contributionId + "\"");  // TODO - see EHR-206
+                    respHeaders.setETag("\"" + contributionId + "\"");
```

Comment:
should this not be suffixed?

Reply:
Correct! Contributions do not have versions.